### ISSUE TITLE
Some sensors stats are duplicated

### ISSUE THREAD

**Issue by nicolargo** [nicolargo/glances#1863]
![Selection_006](https://user-images.githubusercontent.com/776747/118627137-be119700-b7cb-11eb-8b15-8c16c61a39af.png)

Output of Sensors plugins (enties also duplicated, so this is not an UI issue):
```
[{'key': 'label',
  'label': 'acpitz 1',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 27},
 {'key': 'label',
  'label': 'acpitz 2',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 29},
 {'key': 'label',
  'label': 'Package id 0',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 40},
 {'key': 'label',
  'label': 'Core 0',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 37},
 {'key': 'label',
  'label': 'Core 1',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 37},
 {'key': 'label',
  'label': 'Core 2',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 37},
 {'key': 'label',
  'label': 'Core 3',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 35},
 {'key': 'label',
  'label': 'Package id 0',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 40},
 {'key': 'label',
  'label': 'Core 0',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 37},
 {'key': 'label',
  'label': 'Core 1',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 37},
 {'key': 'label',
  'label': 'Core 2',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 37},
 {'key': 'label',
  'label': 'Core 3',
  'type': 'temperature_core',
  'unit': 'C',
  'value': 35},
 {'key': 'label',
  'label': 'sda',
  'type': 'temperature_hdd',
  'unit': 'C',
  'value': 41.0},
 {'key': 'label',
  'label': 'sdb',
  'type': 'temperature_hdd',
  'unit': 'C',
  'value': 39.0}]
```

But perhaps a PSutil one because:

```
$ sensors
acpitz-virtual-0
Adapter: Virtual device
temp1:        +27.8°C  (crit = +119.0°C)
temp2:        +29.8°C  (crit = +119.0°C)

coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +40.0°C  (high = +80.0°C, crit = +100.0°C)
Core 0:        +37.0°C  (high = +80.0°C, crit = +100.0°C)
Core 1:        +40.0°C  (high = +80.0°C, crit = +100.0°C)
Core 2:        +36.0°C  (high = +80.0°C, crit = +100.0°C)
Core 3:        +35.0°C  (high = +80.0°C, crit = +100.0°C)
```

**Comment by nicolargo** [comment]
Change to be done here: https://github.com/nicolargo/glances/blob/fe29162dceea45bf1e65993980bb90127f940a73/glances/plugins/glances_sensors.py#L306-L320

**Comment by nicolargo** [comment]
Looks like to be corrected in PsUtil 5.8.0:

```
$ python 
Python 2.7.5 (default, Nov 16 2020, 22:23:17) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-44)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> 
>>> import psutil
>>> psutil.sensors_temperatures()
{'acpitz': [shwtemp(label='', current=27.8, high=119.0, critical=119.0), shwtemp(label='', current=29.8, high=119.0, critical=119.0)], 'coretemp': [shwtemp(label='Package id 0', current=40.0, high=80.0, critical=100.0), shwtemp(label='Core 0', current=36.0, high=80.0, critical=100.0), shwtemp(label='Core 1', current=38.0, high=80.0, critical=100.0), shwtemp(label='Core 2', current=39.0, high=80.0, critical=100.0), shwtemp(label='Core 3', current=34.0, high=80.0, critical=100.0)]}
>>> from pprint import pprint
>>> pprint(psutil.sensors_temperatures())
{'acpitz': [shwtemp(label='', current=27.8, high=119.0, critical=119.0),
            shwtemp(label='', current=29.8, high=119.0, critical=119.0)],
 'coretemp': [shwtemp(label='Package id 0', current=41.0, high=80.0, critical=100.0),
              shwtemp(label='Core 0', current=36.0, high=80.0, critical=100.0),
              shwtemp(label='Core 1', current=37.0, high=80.0, critical=100.0),
              shwtemp(label='Core 2', current=35.0, high=80.0, critical=100.0),
              shwtemp(label='Core 3', current=34.0, high=80.0, critical=100.0)]}
```

**Comment by nicolargo** [comment]
PsUtil issue. Upgrade it to 5.8.0 or higher.
